### DISCUSSION TITLE
There is no type table to choose from after uploading images on the education package upload education certificate interface

### DISCUSSION THREAD

**Discussion by axin0X** [community/community#69378] [GitHub Education]
### Hi everyone,

When I applied for an education package, there was no type table to choose from after uploading the image on the proof of education interface
,
Just like the picture below

![？](https://github.com/community/community/assets/40687850/65858b99-2285-4d0a-b782-2ef36e57fcdb)

**Comment by arhamansari11** [comment]
I think its due to the language. use english language instead of other languages

**Reply by mantasu** [reply]
It seems that it already is translated to English

**Comment by mantasu** [comment]
From what I've experienced, the staff will determine the proof automatically once it's been captured. Either a student ID or a certificate will do fine as long as it has a translation and an expiry date. Don't worry about manually ticking the proof type as there is no such option as discussed [here](https://github.com/orgs/community/discussions/67819).
